apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -485,11 +503,16 @@ private List<Beneficiary> fetchBeneficiariesByContractAndYearMonth(
     String contractCode = coverageId.getValueNotNull();
 
     /*
-     * Workaround for BFD-1057: The `ORDER BY` required on our "find the bene IDs" query (below)
-     * intermittently causes the PostgreSQL query planner to run a table scan, which takes over an
-     * hour in prod. This _seems_ to be only occurring when the query would return no results. (Yes,
-     * this is odd and we don't entirely trust it.) So, when we're on the first page of results or
-     * not paging at all here, we first pull a count of expected matches here to see if there's any
+     * Workaround for BFD-1057: The `ORDER BY` required on our "find the bene IDs"
+     * query (below)
+     * intermittently causes the PostgreSQL query planner to run a table scan, which
+     * takes over an
+     * hour in prod. This _seems_ to be only occurring when the query would return
+     * no results. (Yes,
+     * this is odd and we don't entirely trust it.) So, when we're on the first page
+     * of results or
+     * not paging at all here, we first pull a count of expected matches here to see
+     * if there's any
```

Comment:
This keeps happening.. are people hitting the full file format with the IDE's (shorter) line formatting options? Or is the auto-formatter doing this once a change is made to the file? It creates a lot of noise in PRs.

Reply:
I think it's a vs code issue. Sometimes I forget to switch from vs code to intellij when working on java code